### PR TITLE
Entry by uuid rather than ?as-of

### PR DESCRIPTION
**Pull request by belucid**
This PR moves entries from `as-of` to `uuid` identification as comments/reactions (collectively, "interactions") will be referencing the entries and the `as-of` referencing is influenced by older, out of date technical concerns (keeping every entry edit).

https://trello.com/c/uU1gKShp

Turns out to be a no-op against the web UI (great!)

Review the code. NB: Share update requests still spec entries w/ as-of. Share story creation is due to change quite a bit w/ ML so this is left alone for now. All other entry operations are by UUID.

To test, use the Web UI against this branch to:

Start w/ some stuff in your system so you can check the migration.

After the migration, does all your stuff look the same?

Add a new topic
  edit the entry
  add a new entry
  delete the entry
  
Share an update w/ some topics selected

Any other UI regression tests you want to do are good too.

### CODE REVIEW

**Comment by belucid** [comment]
@bago2k4 Also this lib PR: https://github.com/open-company/open-company-lib/pull/8

File: src/oc/storage/resources/entry.clj [hunk 1]
```diff
@@ -89,21 +90,21 @@
...
-  (db-common/read-resource conn table-name id))
+  (db-common/read-resource conn table-name uuid))
+
+  ([conn org-uuid :- lib-schema/UniqueID board-uuid :- lib-schema/UniqueID topic-slug :- common/TopicSlug uuid :- lib-schema/UniqueID]
+  {:pre [(db-common/conn? conn)]}
+  (first (db-common/read-resources conn table-name :uuid-topic-slug-board-uuid-org-uuid [[uuid topic-slug board-uuid org-uuid]])))
   
-  ([conn uuid-type :- (schema/enum :board :org) uuid :- lib-schema/UniqueID
-    topic-slug :- common/TopicSlug created-at :- lib-schema/ISO8601]
+  ([conn org-uuid :- lib-schema/UniqueID topic-slug :- common/TopicSlug created-at :- lib-schema/ISO8601]
```
Comment (bago2k4) [inline src/oc/storage/resources/entry.clj]: Why before we had entries connected to board uuid or org uuid and now we have them only by org?
Is this change needed by the new entry schema?

Comment (belucid) [inline src/oc/storage/resources/entry.clj]: We still have both. `get-entry` has 3 variants:

- by uuid (this is enough to find it, but not enough to validate the REST API path, mostly meant for us at the REPL)
- by org, board topic and uuid (this is the main REST API retrieval) 

and

- by org, topic-slug, and created-at (this is a bit legacy, but still used for now in update creation, will get changed when we go to ML stories)
